### DISCUSSION TITLE
Ability to tweak icons..

### DISCUSSION THREAD

**Discussion by Aman9das** [zixpo/candybar#53] [Q&A]
You can take a look at https://github.com/Donnnno/Arcticons

We would love it if we could set a variable say iconColor and change the icon color from say white to  red, similar to how the navigation-drawer icons change colour based on theme..

Is it possible to get some functionality like this... developer end to provide custom apk or better at user end..

Also is it possible to set icon request feature to github, we don't really know how to code much, but the current implementation would be to set up a bot.. from email to github

**Comment by sarsamurmu** [comment]
The project got me mildly interested. So how are you generating all the PNG files? If you're using a script, can you link the script?

**Reply by Aman9das** [reply]
Yes gladly!
https://github.com/Donnnno/Arcticons/blob/candybar/other/rasterdark.sh

**Reply by sarsamurmu** [reply]
Can't you just generate the icons in different colors using your scripts?

**Reply by Aman9das** [reply]
Yes we can build custom apks.. thanks for looking into it though .
